src/components/DayPicker.jsx

```diff
@@ -288,24 +288,19 @@ export default class DayPicker extends React.Component {
   }
 
   renderNavigation() {
-    const isVertical = this.isVertical();
+    const {
+      navPrev,
+      navNext,
+    } = this.props;
 
     return (
-      <div className="DayPicker__nav">
-        <span
-          className="DayPicker__nav--prev"
-          onClick={this.handlePrevMonthClick}
-        >
-          {isVertical ? <ChevronUp /> : <LeftArrow />}
-        </span>
-
-        <span
-          className="DayPicker__nav--next"
-          onClick={this.handleNextMonthClick}
-        >
-          {isVertical ? <ChevronDown /> : <RightArrow />}
-        </span>
-      </div>
+      <DayPickerNavigation
+        navPrev={navPrev}
+        navNext={navNext}
+        isVertical={this.isVertical()}
+        handlePrevMonthClick={this.handlePrevMonthClick}
```

Comment:
I've been in the slow business of renaming `handle...` event methods to `on...`. Let's do that here and call the props `onPrevMonthClick` and `onNextMonthClick`